pkg/config/schema/codegen/templates/crdclient.go.tmpl

```diff
@@ -61,7 +61,7 @@ func update(c kube.Client, cfg config.Config, objMeta metav1.ObjectMeta) (metav1
 func updateStatus(c kube.Client, cfg config.Config, objMeta metav1.ObjectMeta) (metav1.Object, error) {
 	switch cfg.GroupVersionKind {
 {{- range .Entries }}
-	{{- if and (not .Resource.Synthetic) (not .Resource.Builtin) (not (eq .StatusType "")) }}
+	{{- if and (not .Resource.Synthetic) (or (not .Resource.Builtin) (eq .Resource.Identifier "Ingress")) (not (eq .StatusType "")) }}
```

Comment:
@howardjohn added this condition to allow adding `Ingress` type only on `updateStatus`, let me know if this is OK or if we should maybe include all `Builtin` resources in here, and also if modifying only `updateStatus` is OK or if you think I should modify the other methods as well.